### DISCUSSION TITLE
Latest Desktop App (2024.3.0) - can't select collections - so creating new entries isn't possible

### DISCUSSION THREAD

**Discussion by kiwi-x** [dani-garcia/vaultwarden#4454] [Q&A]
Hello,

We are using vaulwarden server (docker deployment 1.30.5) with Web 2024.1.2b.

We use the bitwarden app on Windows, Linux and Mac clients.

We have multiple organisations and users are assigned to organization and collections.

But now we have the issue, that you can't create a new entry for a organization in the 
bitwarden desktop app, 'cause in the desktop app there isn't an assigned collection 
in the select box.

When using the vaulwarden web version with a desktop browser - anything works as 
expected.

For my personal vault - anything works as expected, too.

--
kiwi-x

**Comment by stefan0xC** [comment]
That's a bug in the Desktop app, cf. https://github.com/bitwarden/clients/issues/8258
